JavaScript homeworks/homework-4.js

```diff
@@ -0,0 +1,63 @@
+//Array task 1//
+let fruits = [];
+fruits.push("apple", "banana", "orange");
+console.log(fruits.length); // 3
+console.log(fruits); // (3) ['apple', 'banana', 'orange']
+
+
+//Array task 2//
+var numbers = [1, 2, 3, 4, 5];
+var removedNumber = numbers.pop();
```

Comment:
You are declaring `var removedNumber` in line 10 and line 14 you want to use 2 different variables? Or reassign value for existing one?
you don't need `var` second time you already declared variable

Reply:
Agree. Avoid using same variable names. And better use let in case var is not strictly required.
I guess this is what we've discussed on last lesson